src/controller/CHIPDeviceController.cpp

```diff
@@ -194,6 +195,7 @@ CHIP_ERROR DeviceController::Init(NodeId localDeviceId, ControllerInitParams par
 
         mDeviceAddressUpdateDelegate = params.mDeviceAddressUpdateDelegate;
     }
+    Mdns::Resolver::Instance().StartResolver(mInetLayer, kMdnsPort);
```

Comment:
note: you may need to clean this up after https://github.com/project-chip/connectedhomeip/commit/d9de894033128882ba1e8c5f1631badf40cba205 has been merged as that commit starts the resolver on each device address update. With your change it won't be needed, I guess.

Reply:
aha...that PR also explains why my PR is failing build checks now. I actually think it's worthwhile to start the Resolver as part of the init, since commissioners are now going to be responding to _chipd requests. I'll add vivien to the reviewers and merge that in.